src/style.css

```
@@ -0,0 +1,98 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+}
+
+.container {
+  display: flex;
+}
+
+.block {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  height: 300px;
+  color: white;
+  flex-shrink: 0;
+  flex-grow: 1;
+}
+
+.block--1 {
+  background: red;
+}
+
+.block--2 {
+  background: rgb(204, 0, 0);
+}
+
+.block--3 {
+  background: rgb(153, 0, 0);
+}
+
+.block--4 {
+  background: rgb(102, 0, 0);
+}
+
+.block--5 {
+  background: rgb(51, 0, 0);
+}
+
+.block--6 {
+  background: black;
+}
+
+@media (min-width: 751px) {
+  .container {
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
+
+  .block {
+    flex-basis: 33.33%;
+  }
+
+  .block--4 {
+    order: 2;
+  }
+
+  .block--5 {
+    order: 1;
+  }
+}
+
+@media (min-width: 600px) and (max-width: 750px) {
+  .container {
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
+
+  .block {
+    flex-basis: 50%;
+  }
+
+  .block--3 {
+    order: 1;
+  }
+
+  .block--5 {
+    order: 2;
+  }
+
+  .block--6 {
+    order: 3;
+  }
+}
+
+@media (min-width: 300px) and (max-width: 599px) {
```

```suggestion
@media (max-width: 599px) {
```
it will enough as for me 🤔